docs/book/14-Streams.md

```diff
@@ -85,7 +85,7 @@ Java 设计者面临着这样一个难题：现存的大量类库不仅为 Java
 
 比如在 **Random** 中添加更多的方法。只要不改变原有的方法，现有代码就不会受到干扰。
 
-问题是，接口部分怎么改造呢？特别是涉及集合类接口的部分。如果你想把一个集合转换为流，直接向接口添加新方法会破坏所有老的接口实现类。
+一个大的挑战来自于使用接口的库。集合类是其中关键的一部分，因为你想把集合转为流。但是如果你将一个新方法添加到接口，那就破坏了每一个实现接口的类，因为这些类都没有实现你添加的新方法。
```

Comment:
这个原文是什么

差距有点多

Reply:
原文：The big challenge came from libraries that used interfaces. Collection classes are an essential part of this, because you want to convert collections into streams. But if you add a new method to an interface, you break every class that implements your interface but doesn't implement your new method.
原翻译少了很多内容。原翻译少翻、乱翻、按照英语语序直接翻，导致汉语拗口的地方太多了。